sunpy/map/sources/psp.py

```diff
@@ -31,6 +64,16 @@ def processing_level(self):
     def name(self):
         return 'WISPR ' + super().name
 
+    @property
+    def detector(self):
+        detector = self.meta.get('detector', "")
+        if detector == 1:
+            return "Inner"
+        if detector == 2:
+            return "Outer"
+        # Should not happen
+        return detector
```

Comment:
If the only allowed cases are 1 or 2, then this should be strictly enforced, i.e. this should fail if detector does not equal 1 or 2. 

```suggestion
        if detector == 1:
            return "Inner"
        elif detector == 2:
            return "Outer"
        else:
            raise ValueError(f'Unrecognized detector type {detector}')
```